app/Console/Commands/Import/ImportAnalytics.php

```diff
@@ -2,71 +2,67 @@
 
 namespace App\Console\Commands\Import;
 
-use Illuminate\Support\Facades\Storage;
-
-use League\Csv\Reader;
-use League\Csv\Statement;
-
 use App\Models\Collections\Artwork;
 
 class ImportAnalytics extends AbstractImportCommand
 {
 
-    protected $signature = 'import:analytics';
-
-    protected $description = "Import analytic pageviews for artworks ";
+    protected $signature = 'import:analytics
+                            {page? : Page to begin importing from}';
 
-    protected static $filename = 'artwork-pageviews.csv';
+    protected $description = "Import analytic pageviews for artworks";
 
 
     public function handle()
     {
 
-        // Check if the file exists
-        if( !Storage::exists(self::$filename) )
-        {
-            $this->warn('Could not find ' . $this->getCsvPath());
-            return;
-        }
+        $this->api = env('ANALYTICS_DATA_SERVICE_URL');
 
-        // Spoofing this w/ local file for speed
-        $contents = Storage::get(self::$filename);
+        $this->import( 'analytics', Artwork::class, 'artworks', $this->argument('page') ?: 1 );
 
-        $this->csv = Reader::createFromPath( $this->getCsvPath(), 'r' );
+    }
 
-        // What line the header's at
-        $this->csv->setHeaderOffset(5);
+    /**
+     * Save a new model instance given an object retrieved from an external source.
+     *
+     * @param object  $datum
+     * @param string  $model
+     * @param string  $transformer
+     * @param boolean $fake  Whether or not to fill missing fields w/ fake data.
+     *
+     * @return \Illuminate\Database\Eloquent\Model
+     */
+    protected function save( $datum, $model, $transformer )
```

Comment:
Making a mental note to refactor imports to support "update ignore", wherein only existing records get updated. This would also be useful for `import:images`.